locales/en.json

```diff
@@ -331,31 +331,31 @@
     "migration_0008_general_disclaimer": "To improve the security of your server, it is recommended to let YunoHost manage the SSH configuration. Your current SSH setup differs from the recommendation. If you let YunoHost reconfigure it, the way you connect to your server through SSH will change thusly:",
     "migration_0008_port": "• You will have to connect using port 22 instead of your current custom SSH port. Feel free to reconfigure it;",
     "migration_0008_root": "• You will not be able to connect as root through SSH. Instead you should use the admin user;",
     "migration_0008_dsa": "• The DSA key will be turned off. Hence, you might need to invalidate a spooky warning from your SSH client, and recheck the fingerprint of your server;",
-    "migration_0008_warning": "If you understand those warnings and agree to let YunoHost override your current configuration, run the migration. Otherwise, you can also skip the migration - though it is not recommended.",
-    "migration_0008_no_warning": "No major risk indentified concerning overriding your SSH configuration—one can however not be absolutely sure ;)! Run the migration to override it. Otherwise, you can also skip the migration - though it is not recommended.",
+    "migration_0008_warning": "If you understand those warnings and want YunoHost to override your current configuration, run the migration. Otherwise, you can also skip the migration, though it is not recommended.",
+    "migration_0008_no_warning": "Overriding your SSH configuration should be safe, though this can not be promised! Run the migration to override it. Otherwise, you can also skip the migration, though it is not recommended.",
     "migration_0009_not_needed": "This migration already happened somehow… (?) Skipping.",
     "migration_0011_backup_before_migration": "Creating a backup of LDAP database and apps settings prior to the actual migration.",
-    "migration_0011_can_not_backup_before_migration": "The backup of the system before the migration failed. Migration failed. Error: {error:s}",
+    "migration_0011_can_not_backup_before_migration": "The backup of the system could not be completed before the migration failed. Error: {error:s}",
     "migration_0011_create_group": "Creating a group for each user…",
-    "migration_0011_done": "Migration successful. You are now able to manage usergroups.",
-    "migration_0011_LDAP_config_dirty": "It look like that you customized your LDAP configuration. For this migration the LDAP configuration needs to be updated.\nYou need to save your current configuration, reintialize the original configuration by running 'yunohost tools regen-conf -f' and retry the migration",
+    "migration_0011_done": "Migration completed. You are now able to manage usergroups.",
+    "migration_0011_LDAP_config_dirty": "Save your current custom LDAP conguration, and reintialize the original one by running 'yunohost tools regen-conf -f' and retry the migration.",
```

Comment:
(Note that this message is to be replaced by a different one in https://github.com/YunoHost/yunohost/pull/797/commits/96bc95656c6ad29fa59ae337a0f9f4f04c097261 )

Reply:
I left a suggestion for it in https://github.com/YunoHost/yunohost/commit/96bc95656c6ad29fa59ae337a0f9f4f04c097261#r35495941